src/api/app/models/project.rb

```diff
@@ -79,7 +79,7 @@ def autocomplete(search)
 
   has_one :staging, class_name: 'StagingWorkflow', inverse_of: :project
   belongs_to :staging_workflow, inverse_of: :staging_projects
-  has_many :staged_requests, class_name: 'BsRequest', foreign_key: 'staging_project_id'
+  has_many :staged_requests, class_name: 'BsRequest', foreign_key: 'staging_project_id', dependent: :nullify
```

Comment:
``dependent: :nullify`` should be the default.

Reply:
@ChrisBr i tried it in the rails console, it didn't worked without the `dependent: :nullify`

Reply:
Ok, fine then